test/test_automl/automl_utils.py

```diff
@@ -55,3 +56,74 @@ def count_succeses(cv_results):
         [status in ['Success', 'Success (but do not advance to higher budget)']
          for status in cv_results['status']]
     )
+
+
+class AutoMLLogParser(object):
+    def __init__(self, logfile):
+        self.logfile = logfile
+        self.lines = self.parse_logfile()
+
+    def parse_logfile(self):
+        # We care about the [debug/info/...] messages
+        assert os.path.exists(self.logfile), "{} not found".format(self.logfile)
+
+        with open(self.logfile) as f:
+            content = f.readlines()
+        content = [x.strip() for x in content if re.search(r'[\w+]', x)]
```

Comment:
Could you please do:
```
with open(self.logfile) as fh:
    content = [line.strip() for line in fh if re.search(r'[\w+]', line)]
```
which is shorter and easier to read. So this basically checks that a line has only unicode characters?

Reply:
Done!